pages/index.html

```diff
@@ -1,12 +1,29 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8">
+	<link href="/styles/styles.css" rel="stylesheet">
+	<link href="/styles/about.css" rel="stylesheet">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Alexa Coffman's Portfolio</title>
 </head>
-<body>
-  
+
+<body class="container">
+	<header>
+		<h1>
+        Alexa Coffman's Portfolio
+      </h1> </header>
+	<nav class="nav">
+		<ul>
+			<li><a href="index.html">home</a></li>
+			<li><a href="about.html">about</a></li>
+			<li><a href="portfolio.html">portfolio</a></li>
+		</ul>
+	</nav>
+	<section> <img id="lighthouse" alt="Photo of a lighthouse on a cliff in Iceland" src="/images/iceland.jpeg"> </section>
```

Comment:
Ah.  I see why it was upset about this.  

A `section` that just contains a single image doesn't really make much sense.  Ideally you would just have the image here and tweak your styles to accommodate that (using things like [adjacent sibling selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator) or [`nth-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) if you wanted to make it general.

Though honestly the easiest is probably to just add a rule around `img` tags in your CSS.  If you _really_ need a wrapper element here you could also fallback to using a `div`.